tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+SYZDIR=/usr/local/google/home/jiahengh/gopath/src/github.com/google/syzkaller
```

Comment:
Can we make this a relative path, probably using `dirname $0`?  Then it will work automatically.

Reply:
Done. I just used the ./ if that makes sense.